<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDialog</class>
 <widget class="QDialog" name="TestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>130</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stockInfoUpdateButton">
       <property name="text">
        <string>StockInfo Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="movementUpdateButton">
       <property name="text">
        <string>Movement Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="movementBtn">
       <property name="text">
        <string>MovementTest</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
